src/bin/pgcopydb/pgcmd.c

```diff
@@ -402,8 +402,8 @@ pg_dump_db(PostgresPaths *pgPaths,
 		args[argsIndex++] = (char *) snapshot;
 	}
 
-	args[argsIndex++] = "--section";
-	args[argsIndex++] = (char *) section;
+	args[argsIndex++] = "--section=pre-data";
+	args[argsIndex++] = "--section=post-data";
```

Comment:
Please add a comment here covering why we want both the sections in the same file (pg_restore dependency tracking) and why we prefer using these two options rather than `--schema-only`: we want to retain the BLOB metadata in the dump, and this is included in --pre-data, but not in --schema-only.

Reply:
done! :thumbsup: